src/main.rs

```diff
@@ -28,6 +29,9 @@ struct Args {
     #[clap(long, help = "MAC address to enforce for the VM")]
     vm_mac_address: mac_address::MacAddress,
 
+    #[clap(long, arg_enum, help = "type of network to use for the VM: 'nat', 'host' (default: nat)")]
```

Comment:
The help for the possible and the default values can be generated automatically now:

```suggestion
    #[clap(long, arg_enum, help = "type of network to use for the VM", default_value_t=NetType::Nat)]
```

Reply:
Oh right, forgot to remove this list from help. And thank you about default - thought it's using the first value from enum by default...